app/routes.py

```diff
@@ -1,2 +1,275 @@
-from flask import Blueprint
+import requests
+import os
+from flask import request, Blueprint, make_response, jsonify
+
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from datetime import datetime
+
+
+tasks_bp = Blueprint("tasks", __name__, url_prefix= "/tasks")
+
+@tasks_bp.route("", methods=["POST", "GET"])
+def create_task():
+
+    if request.method == "POST":
+        request_body = request.get_json()
+        if "title" not in request_body or "description" not in request_body or "completed_at" not in request_body:
+            return jsonify({"details": "Invalid data"}), 400
+        else:
+            new_task = Task(title=request_body["title"], description=request_body["description"], completed_at=request_body["completed_at"])
+
+            db.session.add(new_task)
+            db.session.commit()
+
+            return make_response({
+                "task": {
+                "id": new_task.task_id,
+                "title": new_task.title,
+                "description": new_task.description,
+                "is_complete": new_task.task_completed()
+                }
```

Comment:
Notice that this code is repeated many places throughout your `routes.py`. Consider moving the logic to build a json response into a instance method on the task model that you could then call here with something like ```new_task.to_json()```.